src/btf.cpp

```diff
@@ -714,6 +714,39 @@ std::set<std::string> BTF::get_all_structs() const
   return structs;
 }
 
+std::set<std::string> BTF::get_all_iters_from_btf(const struct btf *btf) const
```

Comment:
Nit: use `std::unordered_set` for performance since we do not need ordering here

Reply:
fix.